### ISSUE TITLE
Typescript: 'I18n' can only be imported by using a default import

### ISSUE THREAD

**Issue by akuul** [fnando/i18n#89]
## Description

Importing as` import { I18n } from 'i18n-js'; ` throws TS error  `'I18n' can only be imported by using a default import` . Importing as default import ` import I18n from 'i18n-js'; ` throws on new I18n()
`This expression is not constructable. Type 'typeof I18n' has no construct signatures.`

## How to reproduce

Use I18n as described in docs

## What do you expect

Not to throws Typescript error

## Software:

- Ruby version: 2.7.6
- React Native: 0.73.4
- I18n: 4.4.0

**Comment by TheWirv** [comment]
I have a similar problem:
![image](https://github.com/fnando/i18n/assets/4397989/65ae9f01-c8b9-4307-bf3a-1f3a2b35c03f)
I think, #79 fucked up the imports...

**Comment by fnando** [comment]
What does your tsconfig.json file look like? I have a project running with 4.4.0 and it's working with `import { I18n as I18nJS } from "i18n-js";`.

```console
$ cat app/assets/scripts/config/i18n.ts
import { I18n as I18nJS } from "i18n-js";
import translations from "config/translations.json";

export const i18n = new I18nJS(translations);
i18n.defaultLocale = "en";

$ tsc --noEmit

$ echo $?
0
```

Here's mine:

```json
{
  "include": ["./app/assets/scripts"],
  "compilerOptions": {
    "rootDir": ".",
    "baseUrl": "./app/assets/scripts",
    "lib": ["DOM", "es2020"],
    "target": "es2015",
    "moduleResolution": "node",
    "isolatedModules": true,
    "allowSyntheticDefaultImports": true,
    "esModuleInterop": true,
    "resolveJsonModule": true,
  },
}
```

**Comment by TheWirv** [comment]
I made a PR to fix the issue

**Comment by fnando** [comment]
@TheWirv based on your PR, looks like you don't have `esModuleInterop` set to `true`. Is that correct?

**Comment by TheWirv** [comment]
No, my `tsconfig.json` is as follow:

```json
{
  "extends": "@react-native/typescript-config/tsconfig.json",
  "compilerOptions": {
    "types": ["react-native"],
    "experimentalDecorators": true,
    "baseUrl": ".",
    "paths": {
      "@green-notes/*": ["./src/*"]
    }
  }
}
```

And below is a screenshot excerpt from the RN config:
![image](https://github.com/fnando/i18n/assets/4397989/25f01142-c29f-4f59-8f78-c9d4cc3b9196)

**Comment by fnando** [comment]
OK, it makes to define the whole path then.

**Comment by TheWirv** [comment]
Also, the `json` bit is needed anyway. Otherwise, we wouldn't be able to access the base translations at all. :/

**Comment by fnando** [comment]
@TheWirv just made a change on your pr. Can you please double check that everything still works?

**Comment by TheWirv** [comment]
Yep, works perfectly fine still 👌🏼

**Comment by fnando** [comment]
Just released v4.4.1 with the fix for this issue. Thank you all for the fast turnaround. :)

**Comment by TheWirv** [comment]
Thank you too for the fast review and merge! :)
